npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker_spec.rb

```diff
@@ -202,6 +202,32 @@
       expect(WebMock).to have_requested(:get, registry_listing_url).once
     end
 
+    context "with multiple requirements" do
+      let(:dependency) do
+        Dependabot::Dependency.new(
+          name: "preact",
+          version: "0.1.0",
+          package_manager: "npm_and_yarn",
+          requirements: [
+            {
+              requirement: "^0.1.0",
+              file: "package-lock.json",
+              groups: ["dependencies"],
+              source: { type: "registry", url: "https://registry.npmjs.org" }
+            },
+            {
+              requirement: "^0.1.0",
+              file: "yarn.lock",
+              groups: ["dependencies"],
+              source: { type: "registry", url: "https://registry.yarnpkg.com" }
+            }
+          ]
+        )
+      end
+
+      specify { expect { subject }.not_to raise_error(/Multiple sources/) }
```

Comment:
Would this pass? Just thinking of the case if this error message changes and the spec keeps passing

```suggestion
      specify { expect { subject }.not_to raise_error }
```